src/analysis/analyzer.ts

```diff
@@ -89,7 +89,7 @@ export class Analyzer extends AnalyzerGen implements vs.Disposable {
 			// This will include things like Observatory output and some analyzer logging code.
 			message = message.trim();
 			if (!message.startsWith('--- ') && !message.startsWith('+++ ')) {
-				logError({ message: `Unable to parse message (${e}): ${message}` });
+				console.error(`Unable to parse message (${e}): ${message}`);
```

Comment:
The logging output from the driver didn't have any consistent markers we could use. It may be enough to check for .startsWith('{') and .endsWith('}'), and then assume that any parse errors are worth reporting?